tw/pkg/commands/dgrep/dgrep.go

```diff
@@ -55,7 +76,10 @@ func Command() *cobra.Command {
 	cmd.Flags().DurationVarP(&cfg.Timeout, "timeout", "t", DefaultTimeout, "time to wait for logs to appear")
 	cmd.Flags().IntVarP(&cfg.Retry, "retry", "r", 3, "number of times to retry a failed request")
 	cmd.Flags().BoolVarP(&cfg.IgnoreCase, "ignore-case", "i", true, "toggle to ignore case for the match")
-	cmd.Flags().StringArrayVarP(&cfg.Patterns, "regexp", "e", nil, "regular expression to match")
+	cmd.Flags().StringArrayVarP(&cfg.Patterns, "regexp", "e", nil, "regular expression to match (must be present)")
+	cmd.Flags().StringArrayVar(&cfg.NotExpected, "ne", nil, "regular expression that must NOT be present")
+	cmd.Flags().StringArrayVar(&cfg.NotExpectedExclude, "ne-exclude", nil, "exclude specific patterns from --default (only works with --default)")
+	cmd.Flags().BoolVar(&cfg.DefaultErrors, "default", false, "add default error patterns (adds 22 common error patterns)")
```

Comment:
```suggestion
	cmd.Flags().BoolVar(&cfg.DefaultErrors, "default", false, "add default error patterns (adds 13 common error patterns)")
```

Reply:
Let's remove that number or dynamically calculate using `len()`.